public/js/test/integration/todomvc.js

```diff
@@ -55,7 +74,13 @@ function debuggee(callback) {
     // NOTE: we should be returning a promise here.
     // The problem is if, the client pauses we need to handle that
     // gracefully and resume. We did this on the test-server.
-    win.apiClient.evaluate(script);
+    win.apiClient.evaluate(script).then(response => {
+      if (response.exception) {
+        const errorMsg = response.exceptionMessage;
+        const commandInput = response.input;
+        console.error(`${errorMsg}\n For command:\n${commandInput}`);
+      }
+    });
```

Comment:
One thing I was noticing was that the client "debuggee" script would fail silently when trying to click on an element or type. Adding early errors to the script and logging them in the tests makes it easier to debug.